man/kernel-install.xml

```diff
@@ -176,11 +177,12 @@
         </listitem>
       </varlistentry>
       <varlistentry>
-        <term><command>inspect [<replaceable>KERNEL-IMAGE</replaceable>]</command></term>
+        <term><command>inspect [<replaceable>KERNEL-IMAGE</replaceable>] [<replaceable>KERNEL-VERSION</replaceable>]</command></term>
```

Comment:
this should be:

```
inspect [<replaceable>KERNEL-IMAGE</replaceable> [<replaceable>KERNEL-VERSION</replaceable>]]
```

to communicate clearly that if only specify one string it's the image, not the version

Reply:
the initrd thing is not documented

Reply:
The code actually does 'inspect kernel-version kernel-image', so it should be:
`inspect [[KERNEL-VERSION] KERNEL-IMAGE]'. 
(Keeping the same order of version and image arguments as in `add` is reasonable.)